homeworks/Ivan.Artemenko_IvanArtemenko/homework_2/homework_2_2.js

```diff
@@ -0,0 +1,21 @@
+const readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+rl.question('Enter the first number from 500 to 1000 ', (number_1) => {
+	rl.question('Enter the second number from 500 to 1000 ', (number_2) => {
```

Comment:
if you ask user input, make sure to add some validation of what user enters in the prompt